pubspec.yaml

```diff
@@ -21,13 +22,13 @@ dev_dependencies:
   test: ^1.3.0
 
 flutter:
+  uses-material-design: true
   assets:
-  - assets/
-
+    - assets/
   fonts:
-  - family: Roboto
-    fonts:
-    - asset: assets/fonts/Roboto-Regular.ttf
-  - family: MaterialIcons
-    fonts:
-    - asset: assets/fonts/MaterialIcons-Regular.ttf
+    - family: Roboto
+      fonts:
+        - asset: assets/fonts/Roboto-Regular.ttf
+    - family: MaterialIcons
+      fonts:
+        - asset: assets/fonts/MaterialIcons-Regular.ttf
```

Comment:
are these tabs or spaces?

Reply:
I have no Idea 😅, I didn't made any changes other than the original file.